src/System.Drawing.Common/ref/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp2.0;
+      netfx;
```

Comment:
Do we even care about netfx if this isn't exposing a portable ref?  Wouldn't netfx just be a full facade?  We don't need any facade if there is no binary-portability story for assemblies with a S.D.Common ref.